src/main/java/nextstep/step2/domain/LottoPurchase.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.step2.domain;
+
+public class LottoPurchase {
+	private static final String LOTTO_PURCHASE_MESSAGE = "%d개를 구매했습니다.";
+	public static int LOTTO_PRICE = 1000;
```

Comment:
```suggestion
	public static int LOTTO_PRICE = 1_000;
```
이런식으로 표현하면 숫자가 길어질수록 가독성에 유리합니다!

Reply:
97780a33b7fdf02b9c6576cc5acc5da7db50dd3a 에서 반영하였습니다.